README.md

```diff
@@ -0,0 +1,136 @@
+# twilio-python
+
+[![Build Status](https://secure.travis-ci.org/twilio/twilio-python.png?branch=master)](http://travis-ci.org/twilio/twilio-python)
+[![PyPI](https://img.shields.io/pypi/v/twilio.svg)](https://pypi.python.org/pypi/twilio)
+[![PyPI](https://img.shields.io/pypi/pyversions/twilio.svg)](https://pypi.python.org/pypi/twilio)
+
+A module for using the Twilio REST API and generating valid
+[TwiML](http://www.twilio.com/docs/api/twiml/ "TwiML -
+Twilio Markup Language").
+
+## Recent Update
+
+As of release 6.5.0, Beta and Developer Preview products are now exposed via
+the main `twilio-python` artifact. Releases of the `alpha` branch have been
+discontinued.
+
+If you were using the `alpha` release line, you should be able to switch back
+to the normal release line without issue.
+
+If you were using the normal release line, you should now see several new
+product lines that were historically hidden from you due to their Beta or
+Developer Preview status. Such products are explicitly documented as
+Beta/Developer Preview both in the Twilio docs and console, as well as through
+in-line code documentation here in the library.
+
+## Installation
+
+Install from PyPi using [pip](http://www.pip-installer.org/en/latest/), a
+package manager for Python.
+
+    pip install twilio
```

Comment:
Any reason to not format these like: 
`pip install twilio` or 
```bash
pip install twilio
```

Reply:
No reason. I just restored the old version of this file (and added a couple things that were in the _rst_ variant): https://github.com/twilio/twilio-python/blob/320e3913cce3cc50694d28378d598b0444abe802/README.md

Reply:
I would say to be consistent (e.g. [PHP](https://github.com/twilio/twilio-php)) we should use the pretty code formatting stuff.

Reply:
Hmm, but in the preview it still looks good. (shrug)